### DISCUSSION TITLE
view browser logs & debug console in anthias?

### DISCUSSION THREAD

**Discussion by traeu** [Screenly/Anthias#1881] [Q&A]
Is it possible to view the log and debug console of the browser running in anthias?
I have a strange problem with a little selfmade and locally hosted HTML-site displaying a schedule. It worked before on anthias and on desktop browsers, but suddenly, without a change to the HTML-code or to the anthias-devices, on the same day all anthias raspis stopped displaying parts of the site. Desktop browsers still display the page completely and without problems.
I tried updating anthias and even installing a completely new SD card with it (tried bookworm and bullseye), but the problem persists.

Now it would be helpful to see the log files of the browser inside the anthias-viewer-container. Maybe this could help me to understand why the page is not displayed correctly.

Is there any way to display the browser console right on the screen? Or is it possible to redirect the browser's console output to a file?

**Comment by nicomiguelino** [comment]
Displaying and/or redirecting logs inside a Docker container is not supported at the moment, but Anthias is open to such changes.
If it's possible (and not for private use), can you provide the HTML-site that you're using?

**Comment by traeu** [comment]
I can provide you the website files, but I wouldn't like to post them in public. Is there a way to provide them to you in private?
Alternatively, I can temporarily allow you direct access to the website if you send me your public ip address.

**Reply by nicomiguelino** [reply]
You can send me the files to my email, [nicomiguelino2014@gmail.com](mailto:nicomiguelino2014@gmail.com). Thanks.
